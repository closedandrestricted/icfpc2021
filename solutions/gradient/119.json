{"vertices":[[46,136],[48,143],[49,47],[50,125],[50,94],[51,131],[52,49],[53,138],[53,98],[54,116],[55,107],[54,92],[55,144],[55,77],[56,130],[57,96],[57,66],[56,48],[58,134],[59,139],[59,95],[59,46],[60,117],[61,106],[60,50],[61,93],[62,116],[62,107],[62,97],[62,73],[62,68],[64,128],[64,74],[65,67],[65,52],[67,114],[67,106],[67,97],[67,94],[67,50],[68,121],[69,43],[70,74],[71,68],[73,53],[74,89],[74,46],[77,97]]}