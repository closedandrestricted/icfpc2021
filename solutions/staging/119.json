{"vertices":[[25,72],[70,121],[73,49],[29,68],[70,34],[27,79],[52,132],[55,42],[64,60],[56,64],[47,63],[85,44],[50,25],[46,84],[59,46],[69,71],[42,96],[21,74],[48,74],[26,92],[8,91],[41,94],[47,53],[55,97],[38,98],[74,42],[39,97],[52,24],[47,107],[71,93],[34,76],[74,61],[13,75],[34,94],[93,113],[31,72],[34,57],[58,118],[86,10],[85,19],[11,64],[57,115],[39,84],[20,58],[24,72],[24,89],[56,94],[70,53]]}