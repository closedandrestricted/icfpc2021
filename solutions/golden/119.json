{"vertices":[[45,139],[47,146],[48,50],[49,128],[49,97],[50,134],[50,47],[52,141],[52,101],[53,119],[53,110],[53,95],[54,147],[54,80],[55,133],[56,99],[56,69],[56,50],[57,137],[58,142],[58,98],[58,47],[59,120],[59,109],[59,53],[60,96],[61,119],[61,110],[61,100],[61,76],[61,71],[63,131],[63,77],[64,70],[64,55],[66,117],[66,109],[66,100],[66,97],[66,53],[67,124],[68,46],[69,77],[70,71],[72,56],[73,92],[73,49],[76,100]]}